README.md

```diff
@@ -8,20 +8,104 @@
 Provide all the tools traders need, both professional and hobbyist alike, to create automated trading bots on the
 GDAX and supported digital asset exchanges. Note: Node 7.6 or above is required.
 
+## Getting Started
+### Install Prereqs
```

Comment:
The readme does not need to be as detailed as this is. Especially the VS Code section is nice, but perhaps a bit much here.